src/main/java/com/whiteclarkegroup/liquibaselinter/report/ConsoleReporter.java

```diff
@@ -1,87 +1,81 @@
 package com.whiteclarkegroup.liquibaselinter.report;
```

Comment:
Seems like there have been some changes to the output vs the original here. Are you able to provide a screenshot example of the new output?

Reply:
Sure. All I'm adding is a summary of rule counts in the console and markdown reporters. 

![image](https://user-images.githubusercontent.com/263089/94863709-de414100-0408-11eb-8279-96667e199605.png)

Reply:
Cool that looks good (except for the passed count which we've talked about removing elsewhere)

Reply:
Cool that looks good (except for the passed count which we've talked about removing elsewhere)